code/datums/diseases/kingstons.dm

```diff
@@ -40,7 +40,58 @@
 													"<span class='userdanger'>You cough up a hairball!</span>")
 					affected_mob.Stun(5)
 				else
-					affected_mob.visible_message("<span class='danger'>[affected_mob] form contorts into something more feline!!</span>", \
+					affected_mob.visible_message("<span class='danger'>[affected_mob] form contorts into something more feline!</span>", \
 													"<span class='userdanger'>YOU TURN INTO A TAJARAN!</span>")
-					var/mob/living/carbon/human/catface = affected_mob
-					catface.set_species("Tajaran")
\ No newline at end of file
+					if(!affected_mob.get_species() != "Tajaran")
+						var/mob/living/carbon/human/catface = affected_mob
+						catface.set_species("Tajaran")
+
+
+/datum/disease/kingstons/advanced
+	name = "Advanced Kingstons Syndrome"
+	max_stages = 4
+	spread_text = "Airborne"
+	cure_text = "Plasma"
+	cures = list("plasma")
+	cure_chance = 50
+	agent = "AMB45DR Bacteria"
+	viable_mobtypes = list(/mob/living/carbon/human)
+	permeability_mod = 0.75
+	desc = "If left untreated the subject will mutate to a diffrent species."
+	severity = MEDIUM
+	var/list/virspecies = list("Human", "Tajaran", "Unathi", "Skrell", "Grey")//no karma races sorrys.
+	var/list/virsuffix = list("pox", "rot", "flu", "Cough", "-gitis", "cold", "Rash")
+	var/chosentype
+	var/chosensuff
+
+/datum/disease/kingstons/advanced/New()
+	chosentype = pick(virspecies)
+	chosensuff = pick(virsuffix)
+
+	name = "[chosentype] [chosensuff]"
+
+/datum/disease/kingstons/advanced/stage_act()
+	..()
+	switch(stage)
+		if(1)
+			if(prob(10))
+				to_chat(affected_mob, "<span class='notice'>You feel awkward.</span>")
+		if(2)
+			if(prob(10))
+				to_chat(affected_mob, "<span class='danger'>You itch.</span>")
+		if(3)
+			if(prob(10))
+				to_chat(affected_mob, "<span class='danger'>Your skin starts to flake!</span>")
```

Comment:
`flake` -> `snowflake` 🤣

Reply:
that wasn't even my intent but now that you have that thought in my head...

Reply:
your skin starts to :snowflake: